public/views/index-template.html

```diff
@@ -199,14 +197,18 @@
         </p>
         <p>
           1. This could be caused by your reverse proxy settings.<br /><br />
-          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not using a reverse proxy make sure to set serve_from_sub_path to true.<br />
+          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not
+          using a reverse proxy make sure to set serve_from_sub_path to true.<br />
```

Comment:
```suggestion
          use a reverse proxy to set serve_from_sub_path to true.<br />
```

Reply:
the suggestion has a different meaning which is incorrect.